src/Miunie.Discord/MiunieDiscord.cs

```diff
@@ -1,21 +1,20 @@
-﻿using DSharpPlus.EventArgs;
-using Miunie.Core;
+﻿using Miunie.Core;
 using Miunie.Discord.Logging;
 using System;
 using System.Threading;
 using System.Threading.Tasks;
 using Miunie.Core.Logging;
-using DSharpPlus.Entities;
+using Discord;
 
 namespace Miunie.Discord
 {
     public class MiunieDiscord : IMiunieDiscord
     {
         public string GetBotAvatarUrl()
-            => _discord.Client?.CurrentUser?.AvatarUrl;
+            => _discord.Client?.CurrentUser?.GetAvatarUrl();
 
-        private ConnectionState _connectionState;
-        public ConnectionState ConnectionState
+        private Core.ConnectionState _connectionState;
```

Comment:
Maybe a rename here to something like MiunieConnectionState. Would make a little more sense seeing as it is Miunies connection state.